File: 2023/java/src/main/java/org/frieder/aoc/day17/b/Solution17B.java
package org.frieder.aoc.day17.b;

import org.frieder.aoc.day17.lib.Solver;

import java.io.IOException;

public class Solution17B {

    public static double getResult(String path) throws IOException {
        return new Solver().getResult(path,4,10);
    }
}

File: 2023/php/src/day17/b/Solution17B.php
<?php declare(strict_types=1);

namespace day17\b;

use day17\lib\Solver;
use Monolog\Logger;

final class Solution17B
{
    public static function getResult(string $inputFile, Logger $logger): int
    {
        return (new Solver())->getResult($inputFile, 4, 10);
    }
}